pkg/k8s/apps/translate.go

```diff
@@ -139,7 +139,7 @@ func translate(t *model.Translation, c *kubernetes.Clientset, isOktetoNamespace
 		TranslateOktetoDevSecret(&t.K8sObject.GetPodTemplate().Spec, t.Name, rule.Secrets)
 		if rule.IsMainDevContainer() {
 			TranslateOktetoBinVolumeMounts(devContainer)
-			TranslateOktetoInitBinContainer(rule.InitContainer, &t.K8sObject.GetPodTemplate().Spec)
+			TranslateOktetoInitBinContainer(rule, &t.K8sObject.GetPodTemplate().Spec)
 			TranslateOktetoInitFromImageContainer(&t.K8sObject.GetPodTemplate().Spec, rule)
```

Comment:
Should we do it for `TranslateOktetoInitFromImageContainer` too?

Reply:
It's already implemented here: 
https://github.com/okteto/okteto/blob/948c2ec9026214875dbd7e803f64c84821e005bc/pkg/k8s/apps/translate.go#L634